### PR TITLE
Explain prescription

### PR DESCRIPTION
**Pull request by ijmitch**
Add a new statement to the list of aims and assumption to explain the focus is primarily on the principles and not the tools.

### CODE REVIEW

File: docs/the-git-based-process-you-need-for-mainframe-development-DRAFT/introduction.md [hunk 1]
```diff
@@ -32,6 +32,11 @@ The target audience of this document are Mainframe DevOps architects and SCM spe
 
 Some aims and assumptions which guided our recommendations include:
 
+- We are focusing on recommending an effective branching strategy and workflow. Beyond relying on `git` and the facilities commonly
+provided by hosted `git` service providers, we make few assumptions about the tools chosen  - except, obviously IBM Dependency-Based Build 
+and IBM UrbanCode Deploy.
```
Comment (dennis-behm) [inline docs/the-git-based-process-you-need-for-mainframe-development-DRAFT/introduction.md:37]: We also need to list **IBM Wazi Deploy** as a deployment technology.

Comment (ijmitch) [reply docs/the-git-based-process-you-need-for-mainframe-development-DRAFT/introduction.md]: I considered that but unsure if we had anything concrete to say about Wazi Deploy - but I've added it plus included some links for each one we list.

**Comment by ijmitch** [comment]
Someone will complain I hyphenated 'Dependency-Based'... but I hate it without!

**Comment by lauren-li** [comment]
> Someone will complain I hyphenated 'Dependency-Based'

That would be me, even though I agree the hyphen makes sense grammatically. If you can get it added into DBB's official product name, I'll be happy to include it in our documentation, as well. 🙂

**Comment by ijmitch** [comment]
That last commit removed the hyphen

**Pull request closed without merging**